docker/Dockerfile

```diff
@@ -12,9 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.18-bullseye AS build
-
-RUN apt-get update && apt-get install -y --no-install-recommends apt-transport-https sudo
+FROM ubuntu:jammy-20220531 AS build
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gcc \
```

Comment:
Do we need to install gcc?

Reply:
Yeah, one of the builds failed without it. I don't have the exact location but if we remove it we should see where it fails in `make {run,test,local_test}`!